controllers/progressiverollout_controller.go

```diff
@@ -92,11 +94,21 @@ func (r *ProgressiveRolloutReconciler) Reconcile(ctx context.Context, req ctrl.R
 		}
 
 		// Get the Applications to update
-		scheduledApps := scheduler.Scheduler(apps, stage)
+		scheduledApps := scheduler.Scheduler(outOfSyncApps, stage)
```

Comment:
This was using all apps, but I guess we only want to schedule the `outOfSyncApps`?

Reply:
No, it's all the apps so you know how many are in sync and how many you still have to do. If it's not clear we probably need to expand the scheduler documentation.